### PR TITLE
Improved Russian translate

### PR DESCRIPTION
**Pull request by greggubarev**
- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [x] My changes do not use shortened URLs or affiliate links.

### CODE REVIEW

**Comment by RandellDawson** [comment]
@greggubarev Can you fix the merge conflict so we can get a final review? 

Thanks

**Comment by greggubarev** [comment]
> @greggubarev Can you fix the merge conflict so we can get a final review?
> 
> Thanks

done

**Comment by greggubarev** [comment]
@RandellDawson can you review my PR please?

File: curriculum/challenges/russian/03-front-end-libraries/bootstrap/add-font-awesome-icons-to-our-buttons.russian.md [hunk 1]
```diff
@@ -16,21 +16,25 @@ Font Awesome - удобная библиотека иконок. Эти знач
...
 <section id='tests'>
 
 ```yml
 tests:
-  - text: Добавьте <code>i</code>-элемент с классами <code>fas</code> и <code>fa-thumbs-up</code>.
+
+  - text: Добавьте элемент <code>i</code> с классами <code>fas</code> и <code>fa-thumbs-up</code>.
     testString: assert($("i").is(".fas.fa-thumbs-up") || $("span").is(".fas.fa-thumbs-up"));
-  - text: Ваша <code>fa-thumbs-up</code>-иконка должна находиться внутри кнопки "Like".
+  - text: Ваша иконка <code>fa-thumbs-up</code>  должен находиться внутри вашей кнопки «Нравится».
```
Comment (RandellDawson) [inline curriculum/challenges/russian/03-front-end-libraries/bootstrap/add-font-awesome-icons-to-our-buttons.russian.md]: "Like" should not be translated here.

**Review by RandellDawson — CHANGES_REQUESTED** [review]
@greggubarev See my suggestion.

File: curriculum/challenges/russian/03-front-end-libraries/bootstrap/add-font-awesome-icons-to-our-buttons.russian.md [hunk 2]
```diff
@@ -16,21 +16,25 @@ Font Awesome - удобная библиотека иконок. Эти знач
 
 ## Instructions
 <section id='instructions'>
+
```
Comment (RandellDawson) [inline curriculum/challenges/russian/03-front-end-libraries/bootstrap/add-font-awesome-icons-to-our-buttons.russian.md]: Please remove the extra line.
```suggestion

```

File: curriculum/challenges/russian/03-front-end-libraries/bootstrap/add-font-awesome-icons-to-our-buttons.russian.md [hunk 3]
```diff
@@ -16,21 +16,25 @@ Font Awesome - удобная библиотека иконок. Эти знач
...
 Используйте Font Awesome, чтобы добавить <code>thumbs-up</code>-иконку в вашу кнопку "Like" путём вставки в неё элемента <code>i</code> с классами <code>fa</code> и <code>fa-thumbs-up</code>. Убедитесь, что текст "Like" стоит после иконки.
+
```
Comment (RandellDawson) [inline curriculum/challenges/russian/03-front-end-libraries/bootstrap/add-font-awesome-icons-to-our-buttons.russian.md]: Please remove the extra line.
```suggestion

```

**Review by RandellDawson — CHANGES_REQUESTED** [review]

File: curriculum/challenges/russian/03-front-end-libraries/bootstrap/add-font-awesome-icons-to-our-buttons.russian.md (hunk 1 above)
Comment (greggubarev) [reply curriculum/challenges/russian/03-front-end-libraries/bootstrap/add-font-awesome-icons-to-our-buttons.russian.md]: done!

**Review by RandellDawson — APPROVED** [review]

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**